nautobot_floor_plan/migrations/0009_add_custom_label_support.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 4.2.17 on 2024-12-19 20:44
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import nautobot_floor_plan.utils.custom_validators
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("nautobot_floor_plan", "0008_add_axis_step"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="floorplan",
+            name="x_axis_labels",
+            field=models.CharField(default="numbers", max_length=12),
+        ),
+        migrations.AlterField(
+            model_name="floorplan",
+            name="x_axis_step",
+            field=models.IntegerField(
+                default=1, validators=[nautobot_floor_plan.utils.custom_validators.ValidateNotZero(0)]
```

Comment:
What's the difference between `custom_validators.ValidateNotZero` and `utils.validate_not_zero`? Do we really need both?